.github/update_changelog.py

```diff
@@ -14,7 +14,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 
-"""Adds a new empty entry at the top of the changelog.
+r"""Adds a new empty entry at the top of the changelog.
```

Comment:
That is not a regular expression! 😁

I guess Pylint is tripping on the `shell-session` code block in the docstring.

Reply:
@kdeldycke Yes, the linter whined about backslashes.

Reply:
Again, you are right. Let's keep that `r` prefix then.